examples/with-react-intl/server.ts

```diff
@@ -0,0 +1,36 @@
+import {basename} from 'path';
```

Comment:
If the server is now using TypeScript, the next start script with `node --icu-data-dir=node_modules/full-icu server.js` won't work.

Reply:
ah tks for noticing. Fixed